pages/common/cheatshh.md

```diff
@@ -0,0 +1,33 @@
+# cheatshh
+
+> Cheatshh - CLI cheatshheet with customized descriptions, tldr and groups, to look into for your reference.
```

Comment:
```suggestion
> CLI cheatshheet with customized descriptions, tldr and groups, to look into for your reference.
```
[The Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#example-descriptions) recommends to avoid using the page title in the description.

Reply:
It was not actually a typo, i purposefully added it to match with the name. but sure i corrected it